src/models/validators/SpecialistTestsCommonSchemaSubmitted.ts

```diff
@@ -5,6 +5,34 @@ import {
   testTypesCommonSchema,
 } from './CommonSchema';
 
+export const ivaDefectSchema = Joi.object({
+  sectionNumber: Joi.string().required(),
+  sectionDescription: Joi.string().required(),
+  vehicleTypes: Joi.array().items(Joi.string()).required(),
+  euVehicleCategories: Joi.array().items(Joi.string()).required(),
```

Comment:
Should euVehicleCategories have a list of allowed values? This validation will allow anything as long as its a string?

Reply:
I think this is why previously we looked at converting that field to a 'real' enum. If I remember right, it broke a lot of things in VTA/VTM when we made that change in the types package.

Reply:
Adding it to the types definitions did cause chaos, but happy to add it here, in the joi validation!